src/test/java/seedu/address/model/student/PhoneTest.java

```diff
@@ -31,10 +31,12 @@ public void isValidPhone() {
         assertFalse(Phone.isValidPhone("phone")); // non-numeric
         assertFalse(Phone.isValidPhone("9011p041")); // alphabets within digits
         assertFalse(Phone.isValidPhone("9312 1534")); // spaces within digits
+        assertFalse(Phone.isValidPhone("73121534")); // wrong first digit
+        assertFalse(Phone.isValidPhone("124293842033123")); // long phone numbers
```

Comment:
Perhaps we could consider the test case where it starts with 6/8/9, but has >8 digits in total, instead of this current one (or we could create a new one). Not sure how important it will be, but feels good to have!

Reply:
Ok good catch! Have updated the test to include these 3 cases.